src/features/metadata-instance-editor/Instance.js

```diff
@@ -563,6 +563,7 @@ class Instance extends React.PureComponent<Props, State> {
             return (
                 <Tooltip position="top-left" text={metadataLabelEditText}>
                     <PlainButton
+                        aria-pressed={isEditing ? 'true' : 'false'}
```

Comment:
minor: alphabetize the props

Also, does just providing `aria-pressed={isEditing}` work as expected?

Reply:
Done